lib/store.rb

```diff
@@ -0,0 +1,41 @@
+require 'byebug'
```

Comment:
Maybe moving it to `boot.rb` will allow it to be available throughout the application.

Reply:
Not sure how this was resolved - did you move it to `boot.rb` and see the effect of placing it there.
I would much rather have all the `require`'s in a single file as far as possible.